controls/bison_works.rb

```diff
@@ -0,0 +1,25 @@
+title 'Tests to confirm bison works as expected'
+
+plan_origin = ENV['HAB_ORIGIN']
+plan_name = input('plan_name', value: 'bison')
+
+control 'core-plans-bison-works' do
+  impact 1.0
+  title 'Ensure bison works as expected'
+  desc '
+  '
+  plan_installation_directory = command("hab pkg path #{plan_origin}/#{plan_name}")
+  describe plan_installation_directory do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
+  end
+  
+  plan_pkg_ident = ((plan_installation_directory.stdout.strip).match /(?<=pkgs\/)(.*)/)[1]
+  plan_pkg_version = (plan_pkg_ident.match /^#{plan_origin}\/#{plan_name}\/(?<version>.*)\//)[:version]
+  describe command("DEBUG=true; hab pkg exec #{plan_pkg_ident} bison --version") do
```

Comment:
Can you change this command so that:
- It doesn't use debug = true, output is correct without setting debug flag.
- It uses the /bin/bison to check that the binlink works as expected

Example output of `/bin/bison --version`:
`bison (GNU Bison) 3.4.1
Written by Robert Corbett and Richard Stallman.

Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.`

Reply:
Done, thanks.